loadtesting/k6/tests/sendNeon.test.js

```diff
@@ -1,121 +1,44 @@
-const gasPriceErrorCounter = new Counter('gas_price_errors');
-
 const sendNeonRequestTime = new Trend('send_neon_request_time', true);
-const blockNumberRequestTime = new Trend('block_number_request_time', true);
-const nonceRequestTime = new Trend('nonce_request_time', true);
-const gasPriceRequestTime = new Trend('gas_price_request_time', true);
-
-//TO DO: create rpc client for a session not for each user
-const client = ethClient();
 
-//TO DO: before test actions
-export function setup() {
-    console.log('Setup');
-}
+const testConfig = JSON.parse(open('../run_options/options.json'));
+export const options = testConfig;
 
-export default function sendNeonTest(data) {
-    let nonceCounter = 0;
-    let gasPriceCounter = 0;
-    let receipt;
+const transferAmount = randomIntBetween(1, 5);
 
-    let account = createAccount();
+export default function sendNeonTest() {
     const accountReceiver = createAccount();
-    const accountSender = wallet.newWalletKeyFromPrivateKey(account.private_key);
-    fundAccountWithNeon(accountSender.address, 20);
-
-    let nonce;
-    try {
-        const startTime = new Date();
-        nonce = client.getNonce(accountSender.address);
-        const finishTime = new Date();
-        nonceRequests.add(1);
-        nonceRequestTime.add(finishTime - startTime);
-    } catch (e) {
-        console.log('Error: ' + e);
-        nonceErrorCounter.add(1);
-    }
-
-    let gasPrice;
-    try {
-        const startTime = new Date();
-        gasPrice = client.gasPrice();
-        const finishTime = new Date();
-        gasPriceRequests.add(1);
-        gasPriceRequestTime.add(finishTime - startTime);
-    } catch (e) {
-        console.log('Error: ' + e);
-        gasPriceErrorCounter.add(1);
-    }
-
-
-    const sendingNeonStartTime = new Date();
-
-    try {
-        const startTime = new Date();
-        client.blockNumber();
-        const finishTime = new Date();
-        blockNumberRequests.add(1);
-        blockNumberRequestTime.add(finishTime - startTime);
-    } catch (e) {
-        console.log('Error: ' + e);
-        blockNumberErrorCounter.add(1);
-    }
-
-    try {
-        const startTime = new Date();
-        client.getNonce(accountSender.address);
-        const finishTime = new Date();
-        nonceRequests.add(1);
-        nonceRequestTime.add(finishTime - startTime);
-    } catch (e) {
-        console.log('Error: ' + e);
-        nonceErrorCounter.add(1);
-    }
-
-    try {
-        const startTime = new Date();
-        client.gasPrice();
-        const finishTime = new Date();
-        gasPriceRequests.add(1);
-        gasPriceRequestTime.add(finishTime - startTime);
-    } catch (e) {
-        console.log('Error: ' + e);
-        gasPriceErrorCounter.add(1);
-    }
+    const accountSender = createAccount();
+    const client = ethClient(accountSender.private_key);
+    fundAccountFromFaucet(client, accountSender.address, initialAccountBalance);
```

Comment:
maybe it's better to prepare accounts first? airdrop neons to sender and do only transfer in this function? Looks like we test faucet here under load

Reply:
fixed